examples/compiler_errors/mismatched_init_and_post_upgrade_params/test/tests.ts

```diff
@@ -0,0 +1,69 @@
+import { execSync } from 'child_process';
+import { Test } from 'azle/test';
+
+interface ExecSyncError extends Error {
+    status: number;
+    signal: string;
+    output: Buffer[];
+    pid: number;
+    stdout: Buffer;
+    stderr: Buffer;
+}
+
+export function getTests(): Test[] {
+    return [
+        {
+            name: 'Using mismatched params in $init and $postUpgrade causes a compilation-time error',
+            test: async () => {
+                try {
+                    execSync(`npx azle canister`, { stdio: 'pipe' });
```

Comment:
dfx build

Reply:
I'm going to use `dfx deploy` as that also runs `dfx canister create`.